.github/workflows/go.yml

```diff
@@ -56,3 +56,6 @@ jobs:
 
       - name: Check Security (vulnerable dependencies and insecure practices)
         run: make secure
+
+      - name: Check that all included packages have acceptable OSS licenses
```

Comment:
arguably that could be a separate action workflow, to run only once, cleanly separated

Reply:
That's true for the `security` and `lint` as well (I was actually surprised the matrix build and lint steps were in one workflow.)